src/components/Input/Input.styles.js

```diff
@@ -0,0 +1,153 @@
+/* stylelint-disable no-descending-specificity */
+import styled from "styled-components";
+import { constants, themes } from "../../theme";
+
+const errorColor = ({ theme: { themeName } }) => themes[themeName].error.base;
+const white = ({ theme: { themeName } }) => themes[themeName].white.base;
+const primaryColor = ({ theme: { themeName } }) =>
+  themes[themeName].primary.base;
+const gray01 = ({ theme: { themeName } }) => themes[themeName].gray01;
+const gray02 = ({ theme: { themeName } }) => themes[themeName].gray02;
```

Comment:
Do you think it would be worthwhile to create a theme helper that will get a value from the `theme` object given a path?

E.g.
```
// path logic would be a bit more complex than this, but I think this illustrates the idea
const getThemeVal = ({ theme: { themeName } }, path) => themes[themeName][path];
```

Reply:
I think it's a good idea. Please have a look at my implementation.